components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -90,6 +105,13 @@ __attribute__((objc_subclassing_restricted)) @interface MDCActionSheetController
  */
 - (void)addAction:(nonnull MDCActionSheetAction *)action;
 
+/**
+ An object conforming to MDCActionSheetControllerDelegate. When non-nil, the
```

Comment:
Nit: `The object that acts as the delegate of the @c MDCActionSheetController`

This follows [UICollectionViewDelegate](https://developer.apple.com/documentation/uikit/uicollectionview/1618033-delegate) and [UITableViewDelegate](https://developer.apple.com/documentation/uikit/uitableview/1614894-delegate).

Reply:
Done